### PR TITLE
change support links

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
change `@beetoken.com` to `@beenest.com` and change all support links to link to `https://support.beenest.com`

### CODE REVIEW

File: src/components/work/Footer/Footer.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ const Footer = () => (
         <Col sm="3" lg="3" className="mb-4 mb-lg-0">
           <ListGroup className="list-group-borderless" flush>
             <ListGroupItemHeading className="h6 font-weight-semi-bold">Contact</ListGroupItemHeading>
-            <ListGroupItem tag="a" href="mailto:support@beenest.com?subject=Contact Support">
+            <ListGroupItem tag="a" href="https://support.beenest.com/" target="_blank">
               support@beenest.com
```
Comment (tc) [inline src/components/work/Footer/Footer.tsx]: We should change the text too

File: src/components/routes/Hosts/HostsStart/HostsStart.tsx [hunk 2]
```diff
@@ -96,7 +96,7 @@ const hostsStartPageContent = {
   contact: {
     header: 'Questions? Concerns?',
     description: 'We\'re here for you! Get in touch via email at ',
-    email: 'support@beetoken.com',
+    email: 'support@beenest.com.com',
```
Comment (tc) [inline src/components/routes/Hosts/HostsStart/HostsStart.tsx]: double `.com`

Comment (kcvan) [reply src/components/routes/Hosts/HostsStart/HostsStart.tsx]: 👍

File: src/components/work/Footer/Footer.tsx (hunk 1 above)
Comment (kcvan) [reply src/components/work/Footer/Footer.tsx]: changed to just `support`

**Pull request merged**